frontend/src/components/Entity.astro

```diff
@@ -14,19 +14,22 @@ export interface Props {
           <a href="https://en.wikipedia.org/wiki/{{ entity.title|urlencode }}">
             <img
               class="h-full w-full rounded-full"
-              src="/entity/image/{{ image|urlencode }}"
+              src={askama.fake(
+                "/entity/image/{{ image|urlencode }}",
```

Comment:
Just leaving a note here for the future. We should move this url prefix server-side